Release 17.3.0

twstokes opened the pull request:
Cuts version 17.3.0.

twstokes committed:
Update podspec to 17.3.0.

twstokes committed:
Update changelog.

mokagio committed:
Add PR numbers to public features in changelog

mokagio reviewed (approved):
Thanks @twstokes 

I'm going to merge this, tag, and release.

mokagio committed:
Update CocoaPods from 1.15.2 to 1.16.2

This is an attempt to address the 500 error we got from the API when
trying to publish version 17.3.0.

See
https://buildkite.com/automattic/wordpresskit-ios/builds/1908#01934d17-d6b4-43c7-9005-0fcb799f5222

mokagio commented:
Process breadcrumb: I got a server error when attempting to publish 17.3.0.

https://buildkite.com/automattic/wordpresskit-ios/builds/1908#01934d17-d6b4-43c7-9005-0fcb799f5222

Before retrying, I thought I'd update to the latest CocoaPods version. There's nothing in the changelog that suggests there's a fix for the error we run into, but it's usually best to make sure one is on the latest version before investigating failures. 

The publishing build is triggered by pushing a tag. Which mean to retry it we should bump the version again... To keep things simple, and given the only change is a tooling one, I deleted the 17.3.0 tag here on GitHub and will tag the commit with the CP update as the new 17.3.0 then try again.

mokagio commented:
🙄 so it seems like 17.3.0 was actually published despite the server error

![image](https://github.com/user-attachments/assets/db9ba737-24db-4b85-bf1a-e8a605cc65ad)

For what is worth, `bundle exec pod search WordPressKit` did not and still does not show that version at this time.
